pkg/systemd/logs.html

```diff
@@ -67,7 +67,27 @@
 
         <div class="filter text-search">
             <label for="journal-grep" translate>Text</label>
-            <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+            <span class="filter-group">
+                <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+                <div id="logs-help" class="dropdown">
+                  <a role="link" tabindex="0" id="logs-help-toggle" class="dropdown-toggle dropdown-form-control" data-toggle="dropdown">
+                    <span class="pficon pficon-help"></span>
+                  </a>
+                  <div id="logs-help-menu" class="dropdown-menu">
+                    <span translate>The text entry allows searching the full content of the log entries. It accepts strings such as:</span>
+                    <ul>
+                        <li><span translate>qualifiers</span>, <span translate>E.g</span> 'priority', 'identifier', 'services'</li>
+                        <li><span translate>log fields</span>, <span translate>E.g</span> '_EXE=/usr/bin/python'</li>
+                        <li translate>any free-form string, can be combined using regular expressions</li>
+                    </ul>
+                    <a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/reviewing-logs_system-management-using-the-rhel-8-web-console#filtering-logs-in-the-web-console_reviewing-logs" target="blank" rel="noopener noreferrer"><span translate>Learn more</span><i class="fa fa-external-link fa-xs"></i></a>
+                    <span class="journal-cmd">
+                        <pre id="journal-query"></pre>
+                        <button class="copy-cmd" id="journal-cmd-copy"><i class="fa fa-clipboard fa-xs"></i></button>
```

Comment:
I find it a bit irritating that without fields filters the command always ends in `--`. This looks like it's cut off. It's really not (it's just not getting any additional specifiers), but maybe we can only append an option separator if there are actually some extra search terms?